### PR TITLE
Lt 1099 prohibit application editing

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: ui_automation_tests/conftest.py [hunk 1]
```diff
@@ -97,6 +98,11 @@ def standard_application_exists(apply_for_standard_application):
     pass
 
 
+@when('my application has been withdrawn')  # noqa
+def withdrawn_application_exists(manage_case_status):
```
Comment (phlptt) [inline ui_automation_tests/conftest.py]: thinking this manage_case_status variable should have something to do with it being set to withdrawn too

File: ui_automation_tests/fixtures/manage_case.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from pytest import fixture
+
+from shared.tools.utils import get_lite_client
+
+
+@fixture(scope="module")
```
Comment (phlptt) [inline ui_automation_tests/fixtures/manage_case.py]: change this to scope function

File: ui_automation_tests/step_defs/test_edit_standard_application.py [hunk 3]
```diff
@@ -97,3 +98,8 @@ def assert_ref_name(context, driver):
 @then("I see my edited reference number")
 def assert_ref_num(driver):
     assert '12345678' in driver.find_element_by_css_selector('.lite-task-list').text
+
+
+@then('the edit application button is not present')
+def edit_button_not_present(driver):
+    assert len((ApplicationPage(driver).find_edit_application_button())) == 0
```
Comment (phlptt) [inline ui_automation_tests/step_defs/test_edit_standard_application.py]: does this take a long time to run? 
if so add:
```
    driver.set_timeout_to(0)
    assert len((ApplicationPage(driver).find_edit_application_button())) == 0
    driver.set_timeout_to(10)

File: ui_automation_tests/fixtures/manage_case.py (hunk 2 above)
Comment (stephanieboyd) [reply ui_automation_tests/fixtures/manage_case.py]: Changed

File: ui_automation_tests/step_defs/test_edit_standard_application.py (hunk 3 above)
Comment (stephanieboyd) [reply ui_automation_tests/step_defs/test_edit_standard_application.py]: Not that I noticed, but I've added this

File: ui_automation_tests/conftest.py (hunk 1 above)
Comment (stephanieboyd) [reply ui_automation_tests/conftest.py]: Sure, changed it up to `manage_case_status_to_withdrawn`

**Pull request merged**